spec/break_iterator_spec.cr

```diff
@@ -0,0 +1,15 @@
+require "./spec_helper"
+
+describe "ICU::BreakIterator" do
+  it do
```

Comment:
Can you just specify what's tested here ? (you can take a look to the other specs)
_[if you don't have the time to do it, just say it, I'll do that kind of cosmetic changes in a commit after your PR]_